File: src/006/p006.py
def solve() -> int:
    n = 100  # pylint: disable=invalid-name
    sum_of_squares = (n * (n + 1) * (2 * n + 1)) // 6
    square_of_sum = ((n * (n + 1)) // 2) ** 2
    return square_of_sum - sum_of_squares

File: src/006/p006.go
package problem006

import (
	"strconv"

	"github.com/sunilbpandey/project-euler/src/utils/go/intmath"
)

func Solve() string {
	n := 100
	sumOfSquares := n * (n + 1) * (2*n + 1) / 6
	squareOfSum := intmath.Pow(n*(n+1)/2, 2)
	return strconv.Itoa(squareOfSum - sumOfSquares)
}
